### DISCUSSION TITLE
How nodes handle vertex groups?

### DISCUSSION THREAD

**Discussion by JPLost** [bnpr/Malt#139] [Q&A] [answered]
There is the vertex nodes currently, but how does it handle vertex groupings on the node side currently?

**Comment by pragma37** [comment, marked as answer]
There's no access to vertex groups at the moment, only vertex colors (Node Utils - common > surface vertex color).
It shouldn't be too hard to support a limited amount of vertex groups, something like the first 4-8 groups.

**Reply by JPLost** [reply]
Good to know. If someone wants more, they can give feedback then.
